### ISSUE TITLE
CPU bar jumps from the left

### ISSUE THREAD

**Issue by Ryderjj89** [torchgm/ModernWinver#3]
The CPU bar jumps as the % increases/decreases. You can see that it seems like the bar jumps to the right, detaching from the starting point at the left. See the attached video.

https://user-images.githubusercontent.com/1996734/111823572-7d74ca80-88bb-11eb-89d5-2ff7645f4885.mp4

**Comment by torchgm** [comment]
Yep, I'm aware of this lol; not much I can do about it though as it's just how the ModernWPF progress bar works. It's mildly annoying but hey, I'd rather not have to build my own progress bar. I'd recommend you open an issue with [them instead](https://github.com/Kinnara/ModernWpf/) if I don't get around to it

I'll leave this open for a bit just to remind me that I need to go shout at someone else but yeah I can't really fix this myself without a fair bit of effort

**Comment by Ryderjj89** [comment]
Okay cool. I wasnt sure where the issue was coming from but wanted to drop a line about it. It still looks beautiful and we'll just have to go yell at these folks to fix their progress bars lol.
